### PR TITLE
build: broaden ownership to team

### PR DESCRIPTION
**Pull request by andrewseguin**
Assigned ownership per component made more sense in the early days when we each had fully authored components

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,316 +1,16 @@
...
-/.husky/**                                                  @devversion @josephperrott
-/.github/CODEOWNERS                                         @devversion @josephperrott @andrewseguin @jelbourn
-/.github/ISSUE_TEMPLATE/**                                  @andrewseguin @jelbourn
-/.vscode/**                                                 @devversion @josephperrott @mmalerba
-/.ng-dev/**                                                 @devversion @josephperrott
-/goldens/size-test.yml                                      @andrewseguin @mmalerba @crisbeto
-/goldens/**                                                 @andrewseguin @mmalerba @crisbeto
-/src/*                                                      @devversion @josephperrott
-/*                                                          @devversion @josephperrott
+*                                   @angular/components-googlers
```
Comment (josephperrott) [inline .github/CODEOWNERS:1]: I think you actually need to have the base directories you want to include here.

`*` would just refer to the files at the root of the repository

`**` would be everything in the repo, but that would include the other files I believe.

Comment (andrewseguin) [reply .github/CODEOWNERS:1]: I think `*` might be a special case, cause I do see that something like `dir/*` only covers immediate folder files (not nested). Here's what their [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) say

```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```

Comment (josephperrott) [reply .github/CODEOWNERS:1]: oh okay, works for me!

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request merged**